### ISSUE TITLE
Corrupted image

### ISSUE THREAD

**Issue by DiegoDDT** [diglos/pi-gen#14]
Trying to flash my disk with balena etcher but it says the image is corrupted...

**Comment by diglos** [comment]
Hi, did you check the hash of the file?

**Comment by DiegoDDT** [comment]
Yes, I checked the hash and the file is not corrupted but for some reason this is the feedback that I recieve from Balena Etcher. 
I solved it just going directly with the terminal for flashing the SD with the following command: 

sudo dd bs=1m if=path_of_your_image.img of=/dev/rdiskN; sync    

That worked, on the same file that I feeded to Balena Etcher!
